KsApi/models/graphql/adapters/GraphMutationWatchProjectResponseEnvelope+WatchProjectMutation.Data.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import ReactiveSwift
+
+extension GraphMutationWatchProjectResponseEnvelope {
+  /**
+   Map `GraphAPI.WatchProjectMutation.Data` to a `GraphMutationWatchProjectResponseEnvelope`, otherwise return `nil`
+   */
+  static func from(_ data: GraphAPI.WatchProjectMutation.Data) -> GraphMutationWatchProjectResponseEnvelope? {
+    guard let projectFromData = data.watchProject?.project else {
+      return nil
+    }
+    let project = WatchProject.Project(
+      id: projectFromData.id,
+      isWatched: projectFromData.isWatched
+    )
+
+    return GraphMutationWatchProjectResponseEnvelope(watchProject: WatchProject(project: project))
+  }
+
+  /**
+   Return a signal producer containing `GraphMutationWatchProjectResponseEnvelope` or `ErrorEnvelope`
+   */
+  static func producer(from data: GraphAPI.WatchProjectMutation
```

Comment:
Going with the suggestion above, this producer would work for something like `WatchProjectResponseEnvelope`.

Reply:
Yup, globally renaming the namespace in the last commit.